app/assets/javascripts/discourse/app/templates/components/topic-post-badges.hbs

```diff
@@ -1,8 +1,5 @@
-{{#if unread }}
-  &nbsp;<a href={{url}} title={{i18n "topic.unread_posts" count=unread}} class="badge badge-notification unread">{{unread}}</a>
-{{/if}}
-{{#if newPosts}}
-  &nbsp;<a href={{url}} title={{i18n "topic.total_unread_posts" count=newPosts}} class="badge badge-notification new-posts">{{newPosts}}</a>
+{{#if unreadPosts}}
```

Comment:
I grepped for the name of this component in all-the-themes/plugins and I see it and `topic-post-badges.hbr` are referenced in a few themes/plugins. This PR renames a couple of attributes that this component receives so we probably should go through official themes/plugins and update them.